pkg/ebpf/dns_stats.go

```diff
@@ -75,3 +121,20 @@ func (d *dnsStatKeeper) GetAndResetAllStats() map[dnsKey]dnsStats {
 	d.stats = make(map[dnsKey]dnsStats)
 	return ret
 }
+
+func (d *dnsStatKeeper) removeExpiredStates(earliestTs time.Time) {
```

Comment:
We might need to create a new map without the expired entries instead (similar to what we do in the Conntrack code). See https://github.com/golang/go/issues/20135
(If this issue no longer affects go 1.13.8 let me know 😅)